samples/cpp/stitching_detailed.cpp

```diff
@@ -813,7 +817,7 @@ int main(int argc, char* argv[])
             }
             blender->prepare(corners, sizes);
         }
-        else if (!timelapser)
+        else if (!timelapser && timelapse)
```

Comment:
I mean that it not easy to understand old samples with 1k lines for new users like @pglotov, you or me. There are many historical/legacy problems.